src/rust/bitbox02/src/lib.rs

```diff
@@ -37,8 +37,8 @@ pub mod commander;
 pub mod keystore;
 #[cfg_attr(feature = "testing", path = "memory_stub.rs")]
 pub mod memory;
-pub mod password;
 pub mod random;
+pub mod safeinputstring;
```

Comment:
maybe could even be just `input`? then `input::SafeInputString` or `input::SafeString`.

Reply:
Done